log/zap.go

```diff
@@ -11,6 +12,8 @@ type Zap struct {
 	conf zap.Config
 }
 
+const loggerCtxKey = "saltZapLoggerCtxKey"
```

Comment:
should not be using builtin type keys in context.

```golang
typ ctxKey string

var loggerCtxKey = ctxKey("zap_logger")
```

Reply:
resolved